nexus_constructor/model/value_type.py

```diff
@@ -1,20 +1,47 @@
 from typing import Union
 import numpy as np
 
+
+class ValueTypes:
```

Comment:
The filewriter expects "double", "int64", "uint32" and so on: https://github.com/ess-dmsc/kafka-to-nexus/blob/master/documentation/commands.md